### PR TITLE
Enable debug output for dokku global exports in trace

### LINKED ISSUE #1957: RFC: enable debug output for dokku global exports in trace
- Description of problem:

Dokku exports are not constant, `dokku trace on` does not yield custom set variables in stdout.

- Proposal:

Move `[[ $DOKKU_TRACE ]] && set -x` toward the top of the `dokku` file to capture global env vars in trace.

Thoughts? Be a better mind reader?

### PR DESCRIPTION
**Pull request by u2mejc**
Resolves #1957

### CODE REVIEW

File: dokku [hunk 1]
```diff
@@ -2,6 +2,8 @@
 set -eo pipefail
 shopt -s nullglob
 
+[[ $DOKKU_TRACE ]] && set -x
+
```
Comment (josegonzalez) [inline dokku]: How would `dokku trace on` be affected by this?

Comment (u2mejc) [inline dokku]: amending, but the answer was because non dokku user's call the dokku command again with sudo :wink:

**Comment by u2mejc** [comment]
hmm that's a chicken and egg issue looking at it now...

**Comment by u2mejc** [comment]
There we go. The path I stumbled on was for dokkurc. The DOKKU_ROOT path could be set in the dokkurc file, but it's not possible to change the path before calling the file. That's the chicken and egg, but this change doesn't affect that.

**Comment by josegonzalez** [comment]
Looks fine to me, will let @michaelshobbs merge :)

**Pull request merged**